### PR TITLE
Update The Project With Goodies

### PR DESCRIPTION
**Pull request by khalidabuhakmeh**
- Can now `dotnet pack Xalendar.View.csproj` to create NuGet Package
- Readme.md is updated with quick start and screenshots
- Logo and Icon for Xalendar
- `global.json` to constrain to .NET Core app 3.1

 #hacktoberfest

**Note: You will need to create a NuGet account and upload the NuGet Package from running `dotnet pack`**

You can upload and version manually for now.

### CODE REVIEW

**Review by ionixjunior — CHANGES_REQUESTED** [review]
Hi @khalidabuhakmeh. Thank you for your PR! The logo is amazing 🤩

I added some comments in the review. Could you make these adjustments?

File: README.md [hunk 1]
```diff
@@ -1,7 +1,49 @@
-# Xalendar
-Xalendar: a simple CalendarView for Xamarin.Forms.
+# Xalendar [![codecov](https://codecov.io/gh/ionixjunior/Xalendar/branch/master/graph/badge.svg)](https://codecov.io/gh/ionixjunior/Xalendar)
 
-[![codecov](https://codecov.io/gh/ionixjunior/Xalendar/branch/master/graph/badge.svg)](https://codecov.io/gh/ionixjunior/Xalendar)
+![xalander logo](./assets/logo.png)
+
+Xalender is a calendar view for Xamarin.Forms applications.
```
Comment (khalidabuhakmeh) [inline README.md:5]:
```suggestion
![Xalendar logo](./assets/logo.png)

Xalendar is a calendar view for Xamarin.Forms applications.
```

File: README.md [hunk 2]
```diff
@@ -1,7 +1,49 @@
...
+
+![screenshot](./assets/screenshot.png)
+
+## Getting Started
+
+Install Xalander using NuGet with the following command line to your shared library project.
```
Comment (khalidabuhakmeh) [inline README.md:11]:
```suggestion
Install Xalendar using NuGet with the following command line to your shared library project.
```

File: README.md [hunk 3]
```diff
@@ -1,7 +1,49 @@
...
+
+```console
+> dotnet add package Xalender
```
Comment (khalidabuhakmeh) [inline README.md:14]:
```suggestion
> dotnet add package Xalendar
```

File: README.md [hunk 4]
```diff
@@ -1,7 +1,49 @@
...
+```
+
+To use the Xalender component, you'll need to add the following namespace to the top of your XAML views.
```
Comment (khalidabuhakmeh) [inline README.md:17]:
```suggestion
To use the Xalendar component, you'll need to add the following namespace to the top of your XAML views.
```

File: src/Xalendar.View/Xalendar.View.csproj [hunk 5]
```diff
@@ -2,7 +2,19 @@
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
     <LangVersion>8.0</LangVersion>
+    <PackageVersion>1.0.0</PackageVersion>
     <Nullable>enable</Nullable>
+    <PackageId>Xalendar</PackageId>
```
Comment (khalidabuhakmeh) [inline src/Xalendar.View/Xalendar.View.csproj:7]:
```suggestion
    <PackageId> Xalendar</PackageId>
```

File: src/Xalendar.View/Xalendar.View.csproj [hunk 6]
```diff
@@ -2,7 +2,19 @@
...
+    <Title>Xalander - Xamarin.Forms Calendar</Title>
```
Comment (khalidabuhakmeh) [inline src/Xalendar.View/Xalendar.View.csproj:8]:
```suggestion
    <Title>Xalendar - Xamarin.Forms Calendar</Title>
```

File: src/Xalendar.View/Xalendar.View.csproj [hunk 7]
```diff
@@ -2,7 +2,19 @@
...
+    <Authors>Pedro Jesus</Authors>
```
Comment (khalidabuhakmeh) [inline src/Xalendar.View/Xalendar.View.csproj:9]:
```suggestion
    <Authors>Souza Junior</Authors>
```

File: src/Xalendar.View/Xalendar.View.csproj [hunk 8]
```diff
@@ -2,7 +2,19 @@
...
+    <Description>Simple Calendar component for Xamarin.Forms apps. Works cross-platform on Android and iOS.</Description>
+    <Copyright>2020 Pedro Jesus</Copyright>
```
Comment (khalidabuhakmeh) [inline src/Xalendar.View/Xalendar.View.csproj:11]:
```suggestion
    <Copyright>2020 Souza Junior</Copyright>
```

**Comment by khalidabuhakmeh** [comment]
Well... that was embarrassing. I never claimed to be the world's best speller 🤣.

I believe I've resolved all the errors. Feel free to `Squash and merge`

**Comment by ionixjunior** [comment]
@khalidabuhakmeh haha nice, thank you!

I'll merge it at Thursday during my next live streaming. I'm developing this library every Thursdays at 7 o'clock PM EDT in my YouTube and Twitch channel. Feel free to join us! 

https://www.youtube.com/ionixjunior
https://www.twitch.tv/ionixjunior

**Review by ionixjunior — APPROVED** [review]
Nice! Thank you!

**Pull request merged**